frontend/app/src/pages/user/create/index.vue

```diff
@@ -1,21 +1,142 @@
 
 const { upgradeVisible } = storeToRefs(useSessionAuthStore());
 const { navigateToUserLogin } = useAppNavigation();
 const { createNewAccount, error, loading } = useAccountManagement();
+
+const { t } = useI18n();
+const css = useCssModule();
+
+const step: Ref<number> = ref(1);
+const steps = [
+  {
+    title: t('create_account.steps.step_1.title'),
+    description: t('create_account.steps.step_1.description')
+  },
+  {
+    title: t('create_account.steps.step_2.title'),
+    description: t('create_account.steps.step_2.description')
+  },
+  {
+    title: t('create_account.steps.step_3.title'),
+    description: t('create_account.steps.step_3.description')
+  },
+  {
+    title: t('create_account.steps.step_4.title')
+  }
+];
 </script>
 
 <template>
-  <Fragment>
-    <UpgradeProgressDisplay v-if="upgradeVisible" />
-    <CreateAccountForm
-      v-else
-      :loading="loading"
-      :error="error"
-      @error:clear="error = ''"
-      @cancel="navigateToUserLogin()"
-      @confirm="createNewAccount($event)"
-    />
-  </Fragment>
+  <div :class="css.overlay">
+    <div :class="css.overlay__scroll">
+      <section :class="css.section">
+        <div :class="css.container">
+          <div :class="css.wrapper">
+            <div class="pb-4" data-cy="account-management">
+              <UserHost>
+                <UpgradeProgressDisplay v-if="upgradeVisible" />
+                <CreateAccountWizard
+                  v-else
+                  :step.sync="step"
+                  :loading="loading"
+                  :error="error"
+                  @error:clear="error = ''"
+                  @cancel="navigateToUserLogin()"
+                  @confirm="createNewAccount($event)"
+                />
+              </UserHost>
+            </div>
+          </div>
+          <footer :class="css.container__footer">
+            <div class="w-[400px] mx-auto">
+              <RuiFooterStepper
+                :model-value="step"
+                :pages="steps.length"
+                variant="pill"
+              />
+            </div>
+            <div class="mt-8 flex justify-between lg:justify-end items-center">
+              <span class="lg:hidden" :class="css.footer__text">
+                {{ t('app.copyright', { year: new Date().getFullYear() }) }}
+              </span>
+              <div class="ml-4">
+                <AdaptiveFooterButton />
+              </div>
+            </div>
+          </footer>
+        </div>
+      </section>
+      <span class="hidden lg:block border-l" />
+      <section :class="[css.section, css.section__aside]">
+        <div class="p-12">
+          <div class="mb-10">
+            <RuiLogo class="!h-8" text />
+          </div>
+          <div>
+            <RuiStepper
+              custom
+              orientation="vertical"
+              :step="step"
+              :steps="steps"
+            />
+          </div>
+          <DockerWarning class="mt-8" />
```

Comment:
see the comment for the warning from before, if we v-if here the component won't even be instantiated